src/components/icon/list.tsx

```diff
@@ -1535,16 +1535,16 @@ export function getIcon(
     case IconName.models:
       return (
         <svg
-          width={size}
-          height={size}
+        width={size}
+        height={size}
           viewBox="0 0 24 24"
           fill="none"
           xmlns="http://www.w3.org/2000/svg"
         >
           <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M7 9.5C7.82843 9.5 8.5 8.82843 8.5 8C8.5 7.17157 7.82843 6.5 7 6.5C6.17157 6.5 5.5 7.17157 5.5 8C5.5 8.82843 6.17157 9.5 7 9.5ZM7 11C8.03353 11 8.94499 10.4774 9.48451 9.68196L14.0524 11.4388C14.018 11.6206 14 11.8082 14 12C14 12.2228 14.0243 12.44 14.0704 12.649L9.38065 15.1742C8.83226 14.4602 7.96988 14 7 14C5.34315 14 4 15.3431 4 17C4 18.6569 5.34315 20 7 20C8.65685 20 10 18.6569 10 17C10 16.8511 9.98915 16.7046 9.96818 16.5615L14.7514 13.9859C15.3011 14.6078 16.1047 15 17 15C18.6569 15 20 13.6569 20 12C20 10.3431 18.6569 9 17 9C16.0739 9 15.2459 9.41961 14.6955 10.0791L9.98814 8.26854C9.99599 8.18008 10 8.09051 10 8C10 6.34315 8.65685 5 7 5C5.34315 5 4 6.34315 4 8C4 9.65685 5.34315 11 7 11ZM7 18.5C7.82843 18.5 8.5 17.8284 8.5 17C8.5 16.1716 7.82843 15.5 7 15.5C6.17157 15.5 5.5 16.1716 5.5 17C5.5 17.8284 6.17157 18.5 7 18.5ZM18.5 12C18.5 12.8284 17.8284 13.5 17 13.5C16.1716 13.5 15.5 12.8284 15.5 12C15.5 11.1716 16.1716 10.5 17 10.5C17.8284 10.5 18.5 11.1716 18.5 12Z"
+            fill-rule="evenodd"
+            clip-rule="evenodd"
```

Comment:
These properties need to be camelCase since will be rendered as react component

Reply:
done 🐫